packages/framework-core/src/services/token-verifiers/role-based-token-verifier.ts

```diff
@@ -0,0 +1,36 @@
+import { DecodedToken, TokenVerifier, UserEnvelope } from '@boostercloud/framework-types'
+
+export const DEFAULT_ROLES_CLAIM = 'custom:role'
+
+function rolesFromTokenRole(rolesClaim: unknown): Array<string> {
```

Comment:
I'd indicate the nullability explicitly
```suggestion
function rolesFromTokenRole(rolesClaim: unknown | undefined): Array<string> {
```